packages/core/src/fhir-deduplication/resources/procedure.ts

```diff
@@ -0,0 +1,137 @@
+import { CodeableConcept, Procedure } from "@medplum/fhirtypes";
+import { CPT_CODE, CPT_OID, LOINC_CODE, LOINC_OID } from "../../util/constants";
+import {
+  combineResources,
+  fillMaps,
+  getPerformedDateFromResource,
+  pickMostDescriptiveStatus,
+} from "../shared";
+
+const procedureStatus = [
+  "entered-in-error",
+  "completed",
+  "not-done",
+  "preparation",
+  "in-progress",
+  "on-hold",
+  "stopped",
+  "unknown",
+] as const;
+
+export type ProcedureStatus = (typeof procedureStatus)[number];
+
+export const statusRanking = {
+  unknown: 0,
+  "entered-in-error": 1,
+  preparation: 2,
+  "not-done": 3,
+  "in-progress": 4,
+  "on-hold": 5,
+  stopped: 6,
+  completed: 7,
+};
+
+export function deduplicateProcedures(procedures: Procedure[]): {
+  combinedProcedures: Procedure[];
+  refReplacementMap: Map<string, string[]>;
+} {
+  const { proceduresMap, refReplacementMap } = groupSameProcedures(procedures);
+  return {
+    combinedProcedures: combineResources({
+      combinedMaps: [proceduresMap],
+    }),
+    refReplacementMap,
+  };
+}
+
+/**
+ * Approach:
+ * 1 map, where the key is made of:
+ * - vaccineCode
```

Comment:
wrong docstring here

Reply:
be explicit that dropping all non loinc cpt codes?

Reply:
fixed! good catch